src/components/filter/FilterPanel.tsx

```diff
@@ -130,17 +130,33 @@ const FilterPanel = (props: Props) => {
 			</div>
 			<ModalPortal isOpen={showFilter}>
 				<div ref={filterBaseRef} className='filter'>
-					{config.map(rowConfig =>
-						Array.isArray(rowConfig) ? (
-							<div className='filter__compound' key={rowConfig.map(c => c.id).join('-')}>
-								{rowConfig.map(_rowConfig => (
-									<FilterRow rowConfig={_rowConfig} key={_rowConfig.id} />
-								))}
-							</div>
-						) : (
-							<FilterRow rowConfig={rowConfig} key={rowConfig.id} />
-						),
-					)}
+					{config
+						.sort((a, b) => {
```

Comment:
this is not exactly what i meant
i want events filter panel to have this order:
```
['attachedMessageId', 'type', 'name', 'body, 'text', 'status']
```
and for messages
```
['attachedEventIds', 'type', 'body', 'bodyBinary']
```